src/app/app.routing.ts

```diff
@@ -60,6 +60,11 @@ export const allRoutes: Routes = [
         (c) => c.PublicFormComponent,
       ),
   },
+  {
+    path: "admin-ui",
+    loadChildren: () =>
+      import("./core/config-ui/config-ui.module").then((m) => m.ConfigUiModule),
```

Comment:
TODO: naming (in general, not only the route)

admin-ui? config-ui? something else?

Reply:
I like that thought! Let's keep a hard-wired routed module `/admin` (we should probably prevent users to create that route also) and put everything below this with a structure as you suggested.
I didn't use "/admin" originally, as this is still used by the component that offers a random assortment of technical administration tools. But we could move this into the module and handle under the same parent route.

For the module and components (code) names, should we then also go with the "Admin" prefix (AdminModule, AdminEntityComponent, AdminEntityFormComponent, ...)? That would make it easier to distinguish from the Config module (which inits the app from existing config).